.travis.yml

```diff
@@ -82,16 +66,16 @@ matrix:
                SETUP_CMD=''
 
 install:
-
     - git clone git://github.com/astropy/ci-helpers.git
     - source ci-helpers/travis/setup_conda.sh
 
 
 script:
+   - pip install tox tox-pypi-filter --upgrade
+   - pip install -e .
    - $MAIN_CMD $SETUP_CMD
 
 after_success:
-    # If coveralls.io is set up for this package, uncomment the line
-    # below and replace "packagename" with the name of your package.
-    # The coveragerc file may be customized as needed for your package.
-    - if [[ $SETUP_CMD == 'test --coverage' ]]; then coveralls --rcfile='ccdproc/tests/coveragerc'; fi
+    # Upload coverage results.
+    - pip install codecov
+    - codecov
```

Comment:
You could in principle simply the Travis configuration even further as done in https://github.com/astropy/astropy/pull/9726 but if this works for now you could also wait until https://github.com/astropy/astropy/pull/9726 is finalized since it is still a work in progress.

Reply:
I think we are planning to wait -- we want to have a larger discussion about what is in our test matrix.